astropy/units/equivalencies.py

```diff
@@ -177,7 +184,10 @@ def iconverter_photnu_fla(x):
         (photlam, fnu, converter_photlam_fnu, iconverter_photlam_fnu),
         (photlam, photnu, converter_photlam_photnu, iconverter_photlam_photnu),
         (photnu, fnu, converter_photnu_fnu, iconverter_photnu_fnu),
-        (photnu, fla, converter_photnu_fla, iconverter_photnu_fla)
+        (photnu, fla, converter_photnu_fla, iconverter_photnu_fla),
+        (lla, lnu, converter, iconverter), # luminosity
+        (lnu, nulnu, converter_fnu_nufnu, iconverter_fnu_nufnu),
```

Comment:
Should I rename these converter functions since they are actually slightly more general?